### PR TITLE
more update on offline, favorites, and share of food on android

### PR DESCRIPTION
**Pull request by Ayomikuolatunji**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,9 +49,12 @@
     "gatsby-plugin-manifest": "^5.0.0",
     "gatsby-plugin-material-ui": "^4.0.2",
     "gatsby-plugin-mdx": "^5.0.0",
+    "gatsby-plugin-offline": "^6.11.0",
     "gatsby-plugin-seo": "^0.0.2",
     "gatsby-plugin-sharp": "^5.1.0",
     "gatsby-plugin-sitemap": "^6.0.0",
+    "gatsby-plugin-sw": "^1.0.1",
+    "gatsby-plugin-webfonts": "^2.3.2",
```
Comment (hhimanshu) [inline package.json:65]: Why do we need this dependency?

File: gatsby-browser.jsx [hunk 2]
```diff
@@ -1,16 +1,54 @@
...
 import "@fontsource/poppins/600.css";
 import "@fontsource/poppins/700.css";
 import "./src/styles/styles.css";
+import { navigate } from "gatsby";
+
+// const isBrowser = typeof window !== "undefined";
+
+const WrapPageElement = ({ element }) => {
+  useEffect(() => {
+    if (typeof window !== "undefined") {
```
Comment (hhimanshu) [inline gatsby-browser.jsx:17]: Why this code is duplicated below?

**Review by hhimanshu — COMMENTED** [review]
- Remove `src/.DS_Store`

File: gatsby-browser.jsx (hunk 2 above)
Comment (Ayomikuolatunji) [reply gatsby-browser.jsx:17]: It is not duplicated

Comment (Ayomikuolatunji) [reply gatsby-browser.jsx:17]: They are doing different jobs.

**Review by hhimanshu — APPROVED** [review]

**Pull request merged**